src/test/java/org/reactome/release/dataexport/NCBIFileUploaderIT.java

```diff
@@ -7,35 +7,57 @@
 import static org.reactome.release.dataexport.utilities.FTPFileUploaderTestUtils.getFileNamesInFileListings;
 import static org.reactome.release.dataexport.utilities.FTPFileUploaderTestUtils.getITTestPropertiesObject;
 import static org.reactome.release.dataexport.utilities.FTPFileUploaderTestUtils.getNextReactomeReleaseNumber;
-import static org.reactome.release.dataexport.utilities.NCBIFileUploaderTestUtils.AllItemsMatchingAtLeastOneRegex.allItemsMatchingAtLeastOneRegex;
+import static org.reactome.release.dataexport.utilities.NCBIFileUploaderTestUtils.NonEmptyListWithAllItemsMatchingAtLeastOneRegex.allItemsMatchingAtLeastOneRegex;
+import static org.reactome.release.dataexport.utilities.NCBIFileUploaderTestUtils.createNCBITestUploadFile;
 import static org.reactome.release.dataexport.utilities.NCBIFileUploaderTestUtils.getCurrentNCBIGeneFileNamePattern;
 import static org.reactome.release.dataexport.utilities.NCBIFileUploaderTestUtils.getCurrentNCBIProteinFileName;
 
 import java.io.IOException;
 import java.net.URISyntaxException;
+import java.util.Properties;
 
+import org.junit.jupiter.api.AfterAll;
+import org.junit.jupiter.api.BeforeAll;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.mockito.Mockito;
 import org.mockito.MockitoAnnotations;
 
 public class NCBIFileUploaderIT {
+	private static Properties itTestProperties;
+	private static String mockNCBIFilePathString;
 	private NCBIFileUploader ncbiFileUploader;
 
+	@BeforeAll
+	public static void obtainRealConfigurationProperties() throws IOException, URISyntaxException {
+		itTestProperties = getITTestPropertiesObject();
+		mockNCBIFilePathString = createNCBITestUploadFile().toString();
+	}
+
 	@BeforeEach
 	public void initializeNCBIFileUploader() throws IOException {
-		this.ncbiFileUploader = Mockito.spy(NCBIFileUploader.getInstance(getITTestPropertiesObject()));
+		this.ncbiFileUploader = Mockito.spy(NCBIFileUploader.getInstance(itTestProperties));
 		MockitoAnnotations.initMocks(this);
 	}
 
 	@Test
 	public void logsInSuccessfullyToNCBIFTPServer() throws IOException {
-		assertThat(this.ncbiFileUploader.loginToFTPServer(), is(equalTo(true)));
+		String reasonForPotentialFailure = this.ncbiFileUploader.getFtpClientToServer().getReplyString();
+
+		assertThat(
+			reasonForPotentialFailure,
+			this.ncbiFileUploader.loginToFTPServer(),
+			is(equalTo(true))
+		);
```

Comment:
This reads weirdly: "assert that the reason for potential failure is true" :confused: 
Maybe `reasonForPotentialFailure` needs a better name?

Reply:
How about `messageIfTestFails`?

Reply:
But it's a message string, why is it being compared to `true`?

Reply:
The messageString isn't being compared.  `assertTrue` has a 3 argument form that takes a custom message to display if the test fails as the first argument and the 2nd and 3rd arguments are compared.

Reply:
So `assertThat` will print `reasonForPotentialFailure` if `this.ncbiFileUploader.loginToFTPServer() == true`? It feels very awkward to read this.

Reply:
No, `assertThat` will print `reasonForPotentialFailure` if `this.ncbiFileUploader.loginToFTPServer()` is not true - that is if the test fails.